Source/WebKit/UIProcess/API/Cocoa/WKWebsiteDataStore.mm

```diff
@@ -1019,5 +1019,23 @@ -(void)_setBackupExclusionPeriodForTesting:(double)seconds completionHandler:(vo
 #endif
 }
 
+- (NSUUID *)_identifier
+{
+    auto identifier = _websiteDataStore->configuration().identifier();
+    if (!identifier)
+        return nil;
+
+    auto span = identifier->toSpan();
+    return adoptNS([[NSUUID alloc] initWithUUIDBytes:span.data()]).autorelease();
```

Comment:
I think this is wrong because the stored `UUID` is converted with `UUID(NSUUID * nsUUID)`, i.e. from uuid string. It should be easily verified with existing test as in [260026@main](https://commits.webkit.org/260026@main).

Reply:
Ah I see didn't realize that UUID defines an NSUUID conversion. I'll use that instead.

Reply:
Sounds like there is also an issue with the way the endianness of the raw bytes is represented when passing toSpan into the NSUUID constructor. In any case now I've switched to using the operator.